guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: guide
+title: Silent payments
+description: How silent payments impact and improve bitcoin UX
+nav_order: 11
+parent: How it works
+permalink: /guide/how-it-works/silent-payments/
+redirect_from:
+ - /guide/onboarding/creating-a-new-wallet/
+ - /guide/onboarding/first-use/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/silent-payments/silent-payments-header.png
+image_base: /assets/images/guide/how-it-works/silent-payments/carousel-images/
+images_contacts:
+    - file: clipboard-options-add-contacts-or-pay
+      modalImage: clipboard-options-add-contacts-or-pay
```

Comment:
Modal images should be separate, bigger, images.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/1109/commits/ac341d1d28599f83bd4aaf955467f51d00caadf6